packages/salesforcedx-apex-debugger/src/messages/i18n.ts

```diff
@@ -30,5 +30,7 @@ export const messages = {
   streaming_handshake_timeout_text:
     'Streaming API handshake failed due to a socket timeout. Check your connection and try again.',
   hotswap_warn_text:
-    "You can't modify Apex classes or triggers during an Apex Debugger session. Save your changes after you're done debugging."
+    "You can't modify Apex classes or triggers during an Apex Debugger session. Save your changes after you're done debugging.",
+  created_exception_breakpoint_text: 'Created exception breakpoint for %s.',
+  removed_exception_breakpoint_text: 'Removed exception breakpoint for %s.'
```

Comment:
These messages appear in the Debug Console when there's an active debugger session and an exception breakpoint is added/removed.